src/test/java/com/bunq/sdk/model/generated/object/NotificationUrlTest.java

```diff
@@ -0,0 +1,238 @@
+    private static final String GETTER_DRAFT_PAYMENT = "getDraftPayment";
+    private static final String GETTER_MASTER_CARD_ACTION = "getMasterCardAction";
+    private static final String GETTER_MONETARY_ACCOUNT_BANK = "getMonetaryAccountBank";
+    private static final String GETTER_PAYMENT_BATCH = "getPaymentBatch";
+    private static final String GETTER_REQUEST_INQUIRY = "getRequestInquiry";
+    private static final String GETTER_REQUEST_RESPONSE = "getRequestResponse";
+    private static final String GETTER_SCHEDULE_PAYMENT = "getScheduledPayment";
+    private static final String GETTER_SCHEDULE_INSTANCE = "getScheduledInstance";
+    private static final String GETTER_SHARE_INVITE_BANK_INQUIRY = "getShareInviteBankInquiry";
+    private static final String GETTER_SHARE_INVITE_BANK_RESPONSE = "getShareInviteBankResponse";
+
+    /**
+     * Model json paths constants.
+     */
+    private static final String BASE_PATH_JSON_MODEL = "src/test/Recourse/NotificationUrlJsons";
+    private static final String JSON_PATH_MUTATION_MODEL = BASE_PATH_JSON_MODEL + "/Mutation.json";
+    private static final String JSON_PATH_BUNQ_ME_TAB_MODEL = BASE_PATH_JSON_MODEL + "/BunqMeTab.json";
+    private static final String JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL = BASE_PATH_JSON_MODEL +
+            "/ChatMessageAnnouncement.json";
+    private static final String JSON_PATH_DRAFT_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/DraftPayment.json";
+    private static final String JSON_PATH_MASTER_CARD_ACTION_MODEL = BASE_PATH_JSON_MODEL + "/MasterCardAction.json";
+    private static final String JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = BASE_PATH_JSON_MODEL +
+            "/MonetaryAccountBank.json";
+    private static final String JSON_PATH_PAYMENT_BATCH_MODEL = BASE_PATH_JSON_MODEL + "/PaymentBatch.json";
+    private static final String JSON_PATH_REQUEST_INQUIRY_MODEL = BASE_PATH_JSON_MODEL + "/RequestInquiry.json";
+    private static final String JSON_PATH_REQUEST_RESPONSE_MODEL = BASE_PATH_JSON_MODEL + "/RequestResponse.json";
+    private static final String JSON_PATH_SCHEDULE_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledPayment.json";
+    private static final String JSON_PATH_SCHEDULE_INSTANCE_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledInstance.json";
+    private static final String JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = BASE_PATH_JSON_MODEL +
+            "/ShareInviteBankInquiry.json";
+    private static final String JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = BASE_PATH_JSON_MODEL +
+            "/ShareInviteBankResponse.json";
+
+    private static final Type NOTIFICATION_URL_TYPE = new TypeToken<NotificationUrl>(){}.getType();
+
+    private void executeNotificationUrlTest(
+            String expectedJsonFileName,
+            String classNameExpected,
+            String referencedObjectGetterName) throws FileNotFoundException, NoSuchMethodException,
```

Comment:
Multi-line argument formatting should have closing braces on new lines:
```
 private void executeNotificationUrlTest(
   String expectedJsonFileName,
   String classNameExpected,
   String referencedObjectGetterName
) throws FileNotFoundException, NoSuchMethodException,
```

Please fix this throughout the changes as well %)